memory/libhugetlbfs.py

```diff
@@ -1,5 +1,28 @@
 #!/usr/bin/env python
 
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+#
+# See LICENSE for more details.
+#
+# Copyright: 2016 IBM
+# Author: Santhosh G <santhog4@linux.vnet.ibm.com>
+#
+# Based on code by:
+# Author: Yao Fei Zhu <walkinair@cn.ibm.com>
+# copyright : 2006 IBM
```

Comment:
Yes, this one is the first committer and the guy from the `control` file marked as author.